### PR TITLE
Few minor changes

### PR DESCRIPTION
**Pull request by nthndy**
I changed the way the zip files are loaded in notebook B as the path was not correct.
Also changed the way the dictionary of labels was created in notebook C as it relied on the new user having the exact same labels as you, which may not be the case. 
Few more suggestions incoming on the google doc and a few other issues I could not solve but all in all I ran right through the protocol to the end successfully (until the last cell re. UMAP).

### CODE REVIEW

**Review by laureho — APPROVED** [review]
LGTM! @nthndy could you just go to notebook A and under the "Kernel" menu press the "Restart & Clear Output"? Clearing the outputs will make the notebook look neater.

File: notebooks/B_CNN_Data_Preview_Images.ipynb [hunk 1]
```diff
@@ -255,7 +255,7 @@
     "    \n",
     "    # Add counts to label counter:\n",
     "    for lab in counts:\n",
-    "        label_count[lab[0]] += lab[1]\n",
+    "        label_count[lab[0]] = lab[1]\n",
```
Comment (quantumjot) [inline notebooks/B_CNN_Data_Preview_Images.ipynb:258]: I think this change may break @KristinaUlicna's code

Comment (KristinaUlicna) [reply notebooks/B_CNN_Data_Preview_Images.ipynb:258]: I have a whole new notebook to replace notebook B now (uses the cellx loader), will submit a PR with the changes soon! So don't worry about that :-)

**Review by quantumjot — APPROVED** [review]
OK, I'm going to merge this, and let PR #35 fix some of these issues. Thanks @nthndy !

**Pull request merged**